test/Interop/Cxx/static/static-member-var.swift

```diff
@@ -0,0 +1,74 @@
+// Test that static member variables are handled properly by the ClangImporter.
+
+// RUN: %empty-directory(%t)
+// RUN: %target-clang -c %S/Inputs/static-member-var.cc -I %S/Inputs -fPIC -o %t/static-member-var.o
+// RUN: %target-build-swift %s -I %S/Inputs -o %t/statics %t/static-member-var.o -Xfrontend -enable-cxx-interop
+// RUN: %target-codesign %t/statics
+// RUN: %target-run %t/statics 2&>1
+//
+// REQUIRES: executable_test
+
+import StaticMemberVar
+import StdlibUnittest
+
+var StaticsTestSuite = TestSuite("statics-member-var")
```

Comment:
```suggestion
var StaticsTestSuite = TestSuite("StaticsTestSuite")
```

The test suite is usually named after the var.

Reply:
Feel free to use any other name of course.